### ISSUE TITLE
Support for loading custom models

### ISSUE THREAD

**Issue by yhm3ps1jfp76nkd** [Lucchetto/SuperImage#12]
Would be nice if the user could load a custom upscaling/enhancing model, like one from here https://upscale.wiki/wiki/Model_Database

Not sure if all of these are supported by MNN, however

**Comment by Lucchetto** [comment]
Yes, I'd like to work on model conversion and custom loading later
